topics/statistics/tutorials/dome/tutorial.md

```diff
@@ -13,11 +13,574 @@ key_points:
+>
+> > <solution-title></solution-title>
+> >**From Example Publication**  
+> > Protein Data Bank (PDB). X-ray structures missing residues.
+> > *Npos* = 339,603 residues.
+> > *Nneg* = 6,168,717 residues.
+> > Previously used in (Walsh et al., Bioinformatics 2015) as an independent benchmark set.
+> {: .solution}
+{: .question}
+
+
+## 1.2 Dataset Splits
+
+
+Dataset splits refer to the process of dividing a dataset into distinct subsets for different purposes, mainly in machine learning or data science tasks. The most common splits are:
+
+
+- Training Set: This is the largest subset, used to train the machine learning model. The model "learns" from this data by adjusting its internal parameters to minimize prediction errors.
+
+- Validation Set: A separate subset used to fine-tune the model's hyperparameters. The model doesn't learn directly from this data, but it helps monitor the model's performance and avoid overfitting, which is when a model becomes too tailored to the training data and doesn't generalize well.
+
+- Test Set: This is the final subset, used to evaluate the model's performance. The test set remains unseen by the model until after training and validation are complete, providing an unbiased estimate of how well the model generalizes to new, unseen data.
+
+
+
+In addition to these, there are some variations in dataset splitting strategies:
+
+
+
+- Holdout Split: A simple division where a fixed percentage of data is reserved for testing (e.g., 80% training, 20% test).
+
+- Cross-validation: In this technique, the dataset is split multiple times into training and validation sets, ensuring each data point is used for validation at least once (e.g., 5-fold cross-validation). This provides a more robust evaluation of the model’s performance.
+
+
+> <question-title> Key Questions </question-title>
+>
+> How __many data points__ are in the training and test sets?  
+> Was a __separate validation__ set used, and if yes, how large was it?  
+> Are the __distributions__ of data types (*Npos* and *Nneg*) in the training and test sets different? Are the distributions of data types in both training and test sets plotted?
+>
+> > <solution-title></solution-title>
+> >**From Example Publication**  
+> > training set: N/A. *Npos,test* = 339,603 residues. *Nneg,test* = 6,168,717 residues. No validation set. 5.22% positives on the test set.
+> {: .solution}
+{: .question}
+      
+
+
+## 1.3 Redundancy between data splits
+
+
+ Redundancy between data splits occurs when the same data points are present in more than one of the training, validation, or test sets. This is undesirable because it can distort model evaluation and lead to overoptimistic performance metrics (e.g. eliminating data points more similar than X%). 
+This may effect the mfodel by introcuding an overfitting risk, unreliable performance metrics and/or lack of generalization.
```

Comment:
```suggestion
This may impact the model by introducing the risk of overfitting and poor generalization rendering the performance metrics as unreliable.
```